mava/configs/env/mabrax.yaml

```diff
@@ -0,0 +1,15 @@
+# ---Environment Configs---
+defaults:
+  - _self_
+
+# Possible envs: [ant_4x2, halfcheetah_6x1, hopper_3x1, humanoid_9|8, walker2d_2x3]
+env_name: MABRAX
+scenario:
+  name: ant_4x2
+  task_name: ant_4x2  # In this case: env_name = task_name
```

Comment:
If we don't use task_name lets leave it out of this config

Reply:
Removed ✔️